app/javascript/styles/components.scss

```diff
@@ -2653,19 +2653,36 @@ button.icon-button.active i.fa-retweet {
   flex-direction: column;
 }
 
-@keyframes pulse {
-  0% {
-    opacity: 1;
+@keyframes heartbeat {
+  from {
+    transform: scale(1);
+    transform-origin: center center;
+    animation-timing-function: ease-out;
```

Comment:
This new animation is cute, but why do we need to change the `animation-timing-function` in each keyframe? Can't we just set `animation-timing-function: ease-in-out` once here?

Reply:
[This](https://github.com/tootsuite/mastodon/commit/5404ff3995b9215d6ac610ae6b00d351cfdb8d3a) is what I'm talking about; I just tested in Chrome and Firefox and it looks the same to me…

Reply:
@nolanlawson I actually generated this with http://animista.net/play/attention/pulsate